src/TodoFooter.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  filter: string;
+  onChange: (filter: string) => void;
+  activeItems: number;
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  filter,
+  onChange,
+  activeItems,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeItems} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', { selected: filter === 'all' })}
+          data-cy="FilterLinkAll"
+          onClick={() => onChange('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filter === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => onChange('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: filter === 'completed',
+          })}
+          data-cy="FilterLinkCompleted"
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method